tinygrad/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .optim import *
+from .ops import *
```

Comment:
You shouldn't need to import ops, that should only be imported by tensor. And doesn't this make like "tinygrad.SGD" work? It should be "tinygrad.optim.SGD" always.